datafusion/sqllogictest/test_files/array.slt

```diff
@@ -1433,6 +1433,79 @@ NULL 23
 NULL 43
 5 NULL
 
+
+## array_max
+# array_max scalar function #1 (with positive index)
+query I
+select array_max(make_array(5, 3, 6, 4));
```

Comment:
add a test where query process couple arrays in single array_max call


```
SELECT input, array_max(input)
FROM (
  SELECT make_array(d-1, d, d-2)
  FROM (VALUES (0), (10), (20), (30), (NULL)) t(d)
)
```

Reply:
Test coverage is added for this case as well.